code/datums/map_config.dm

```diff
@@ -23,6 +23,8 @@
 
 	var/minetype = "lavaland"
 
+	var/orientation = SOUTH		//oh no. this determines what orientation it'll be loaded in.

```

Comment:
It seems more sensible to have north be the default and have orientation be the side of the map that is the top

Reply:
@ninjanomnom reason it's south is because byond's default placement direction is south, so I had that be the "anchor point" for the default map, but I can make it north if needed.

Reply:
Ah ok that makes sense